### PR TITLE
add showlabels compatibility

### PR DESCRIPTION
**Pull request by mbertucci47**
Lists [showlabels](https://ctan.org/pkg/showlabels) as compatible and adds a test. The labels are correctly tagged as artifacts.

### CODE REVIEW

**Comment by FrankMittelbach** [comment]
I think it is arguable if this is correct. Artifact means that this information should be ignored when presenting the document in a different format. However, one can argue that this is intentional information, ie data dat should be presented, e.g., as an "aside". I think this warrants further thought.

**Comment by FrankMittelbach** [comment]
In other wirds, maybe this should be labeled, partially compatible with a note or task stating that what kind of tagging should be deployed needs further thoughts.

File: _data/tagging-status.yml [hunk 1]
```diff
@@ -8462,13 +8462,12 @@
 
  - name: showlabels
    type: package
-   status: unknown
+   status: compatible
```
Comment (davidcarlisle) [inline _data/tagging-status.yml:8465]: @FrankMittelbach yes someone using show lablels to edit a document would want them read out so probably not an artifact. I note `showkeys` is marked as partially-compatible with a comment querying the tagging status. In the end they should both end up with similar tagging.  perhaps make tis partially-compatible to match.

**Comment by mbertucci47** [comment]
Changed to partially-compatible with a tasks entry. Let me know if anything should be changed or added

**Comment by FrankMittelbach** [comment]
@mbertucci47 since the discussion we had is here and not on any issue that can be easily found, either link the merge record or not in thecomments that these need some decision on the right tagging, thanks

**Comment by nxg** [comment]
I'm the showlabels maintainer, and would be very happy to make suitable adjustments to the package, if someone can give me a suitable pointer, to get me started (@davidcarlisle, _of course_ I'm not going to skip the opportunity to do so, not if `showkeys` does it properly!)

I'm going to be spending some $WORK time on learning about PDF tagging of LaTeX documents, in pursuit of accessibility goals, so I'm doubly motivated here, as a fix-up project such as this would be a very useful on-ramp to the relevant documentation and tooling.

**Comment by u-fischer** [comment]
@nxg well at first the claim above that the labels are tagged as artifacts is wrong. This is only the case if lualatex is used, with pdflatex the labels are untagged and that should probably be changed (or documented that with pdflatex the document is not fully tagged).

I'm not sure if tagging these keys as Aside as suggested above is really a good idea. I don't quite see why e.g. a blind person should use a screen reader to find the labels in a document if they can use an editor and search. And if they do, they probably want that read directly at the relevant structure, e.g. inside the heading and not somewhat later. Personally I think keeping the current status would be ok unless some concrete request for a change comes up. 

But there should be more tests with different variants. Most importantly (as you hook e.g. into `\maketag@@@`) there should be tests with math tagging to ensure that this doesn't break something, so with `\DocumentMetadata{tagging=on,tagging-setup={math/setup=mathml-SE}}`  and unicode-math.

**Pull request merged**